platforms/native/engine/jsb-gfx.js

```diff
@@ -57,7 +57,7 @@ deviceProto.copyTexImagesToTexture = function (texImages, texture, regions) {
 
 const oldDeviceCreateSwapchainFunc = deviceProto.createSwapchain;
 deviceProto.createSwapchain = function (info) {
-    info.windowHandle = jsbWindow.windowHandler;
+    info.windowHandle = window.oh ? jsb.device.getWindowHandle() : jsbWindow.windowHandler;
```

Comment:
Why oh is special?

Reply:
Because openharmony does not support the execution of js scripts in c++